tests/unit/dom/select-files-test.js

```diff
@@ -45,8 +45,8 @@ module('DOM Helper: selectFiles', function(hooks) {
     });
 
     await setupContext(context);
-    await triggerEvent(element, 'change', [textFile]);
-    await triggerEvent(element, 'change', [imageFile]);
```

Comment:
I think its 👍 	to change this test to use the non-deprecated path, but would you mind adding a test specifically about the deprecation (passing options as an array)? I just want to make sure we keep that under test until we actually remove support.

Reply:
On line 68 there is already a test asserting the deprecation. So I figured it was extra safe to change this one.

Reply:
Aha, thank you!